lib/chat/bloc/provider/ChatProvider.dart

```diff
@@ -135,33 +133,26 @@ class ChatProvider implements ViewModelProvider<ChatViewModel> {
         _insertNewMessageToList(
             _chatMessageHandler.getMessageFromEntity(formItem));
         _increaseMessageCount();
-        _updateAvatarVisibility();
+        _updatePreviousMessages();
         _setInteractiveWidget(formItem.inputRequest);
         _notifyController();
       } else {
-        _setSummaryWidget();
+        _setSummaryDetailsButton();
       }
     });
   }
 
-  void _setSummaryWidget() {
-    _chatViewModel.interactiveWidget = CardView(
-      child: Summary(
-        viewModel: _chatSummaryProviderHelper.getSummary(
-          inputModel: _responseMap,
-          titleValue: _Strings.summaryTitleValue,
-          titleText: _Strings.summaryTitleLabel.toUpperCase(),
-          actionText: _Strings.summaryActionButtonText,
-        ),
-        onTap: _onSummaryWidgetActionButtonTap,
-      ),
+  void _setSummaryDetailsButton() {
+    _chatViewModel.interactiveWidget = _chatSummaryProviderHelper.getSummary(
+      title: Intl.message(_Strings.viewDetails()),
```

Comment:
the _Strings.viewDetails() is already an Intl.message
```suggestion
      title: _Strings.viewDetails(),
```

Reply:
Changed, stupid error from my side, sorry for that.

Reply:
np